content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,166 @@
+2019-10-09 14:10:58 UTC | CORE | INFO | (pkg/logs/input/container/launcher.go:58 in NewLauncher) | Could not setup the kubernetes launcher: /var/log/pods not found
+2019-10-09 14:10:58 UTC | CORE | INFO | (pkg/logs/input/container/launcher.go:61 in NewLauncher) | Container logs won't be collected
+```
+
+And this log will appear when the Agent attempts to run the container check:
+
+```
+2019-10-09 14:12:27 UTC | PROCESS | CRITICAL | (collector.go:91 in runCheck) | Unable to run check 'container': temporary failure in detector, will retry later: No collector detected
+```
+
+Relaunch the Agent container with the following option: `-v /var/run/docker.sock:/var/run/docker.sock:ro` to allow access to the Docker socket.
+{{% /tab %}} {{% tab "Host Installation" %}}
+If you're using the Host Agent, the user `dd-agent` needs to be added to the Docker group in order to have permission to read from the Docker socket. If the Agent user isn't added to the Docker user group, Agent Status command will show `Status: Pending` under "Logs Agent". You should also see the following error logs in the Agent log file at `/var/log/datadog/agent.log`:
+
+```
+2019-10-11 09:17:56 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:360 in initListenerCandidates) | docker listener cannot start, will retry: temporary failure in dockerutil, will retry later: could not determine docker server API version: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/version: dial unix /var/run/docker.sock: connect: permission denied
+2019-10-11 09:17:56 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:123 in collect) | Unable to collect configurations from provider docker: temporary failure in dockerutil, will retry later: could not determine docker server API version: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/version: dial unix /var/run/docker.sock: connect: permission denied
+```
+
+ To add the Host Agent to the Docker user group, perform the following command: `usermod -a -G docker dd-agent`.
+{{% /tab %}} {{< /tabs >}}
+
+{{< tabs >}} {{% tab "Container Installation" %}}
+## `container_collect_all` isn't set
+Sometimes when you run the Agent status command the Log Status shows no integrations and you see `LogsProcessed: 0 and LogsSent: 0`.
+
+```
+==========
+Logs Agent
+==========
+    LogsProcessed: 0
+    LogsSent: 0
+```
+
+When you see this, it means that the Agent isn't collecting any log files from your integrations. For the Container Agent, this can happen if you haven't set the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` since without this option the container Agent won't try to collect logs from other containers. To double check what environment variables you've set, run the command `docker inspect <agent-container>`.
```

Comment:
```suggestion
When you see this, it means that the Agent isn't collecting any log files from your integrations. For the Container Agent, this can happen if you haven't set the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL`—since without this option, the container Agent does not try to collect logs from other containers. To check what environment variables you've set, run the command `docker inspect <agent-container>`.
```